.github/ISSUE_TEMPLATE/issue report.yml

```diff
@@ -24,24 +26,14 @@ body:
           4. See error...
     validations:
       required: true
-  - type: dropdown
+  - type: input
     id: version
     attributes:
       label: Jellyfin Version
       description: What version of Jellyfin are you running?
-      options:
```

Comment:
i would rather see this to be expanded to include recent minor versions instead of an input

Reply:
I don't think including recent minor versions would be sustainable.

Reasons as follows:
1. With how long it generally takes for things to get merged it may be months between a new release and the option being available in the issue report template.
2. Including a whole bunch of minor versions will make the dropdown very long and inconvenient 
3. Without an automated mechanism to update it every release, it would have to be done manually

Reply:
i see but the effort then lies on our side, which we can include in the process, push with enough annoying the maintainers with relative ease. If we make it an free-text input the burden of putting something useful there lies on the user-side which will result in more issues/unclarities.

Reply:
If updating the list gets included into the formal release process then I'm ok with having all the subversions in the dropdown. How would the process of adding such thing into the release process work?

Reply:
Annoying @joshuaboniface for as long as it needs so he remembers it.

Reply:
If this can be made a process I'll write up another PR for a more ideal (I think) template for it.